blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,225 @@
+and Michał Żygowski in the [set of patches](https://review.coreboot.org/c/coreboot/+/68449/11)
+has added improvements huge improvements. Also, Michał Żygowski made a significant
+contribution to [Tiger Lake support](https://review.coreboot.org/c/coreboot/+/56171).
+
+In the previous year, our team also added
+[support for MSI](https://review.coreboot.org/c/coreboot/+/68078) PRO Z690-A
+WIFI and MSI PRO Z690-A in DDR4 and DDR5 versions.
+
+Krystian Hebel, Sergii Dmytruk, and Yaroslav Kurlaev have made major contributions
+to developing and maintaining support for Power platforms. Their changes include
+[emulation improvements](https://review.coreboot.org/c/coreboot/+/67061),
+and fixing ongoing issues related to these platforms.
+
+#### Platform security (TPM, TXT)
+
+Our team also worked on [TPM](https://3mdeb.com/shop/modules/tpm-2-0/) and
+[TXT](https://blog.3mdeb.com/2022/2022-03-17-optiplex-txt/) solutions that
+could support the security of coreboot-based platforms. Sergii Dmytruk added
+quite a few improvements related to
+[TPM support](https://review.coreboot.org/c/coreboot/+/68748),
+while Michal Zygowski added some
+[TXT improvements](https://review.coreboot.org/c/coreboot/+/59519).
+
+#### coreboot documentation improvements
+
+The work of our engineers was not closed only to changes in the code.
+It is equally important to make the code readable, clear, and understandable
+to everyone. Hence, it is worth highlighting our efforts to keep the coreboot
+documentation up to date. In this matter, the main contributors were
+[Michał Żygowski](https://review.coreboot.org/c/coreboot/+/65702) and
+[Sergii Dmytruk](https://review.coreboot.org/c/coreboot/+/68752)
+
+If you are interested in improving the security of your devices while reducing
+your dependence on proprietary firmware, you could benefit from using coreboot.
+We can also help you reduce time-to-market by simplifying the firmware
+development process. As well as if you want your firmware to be user-friendly
+and well-documented user-friendly and well-documented then coreboot is for you.
+
+---
+
+![fwupd-logo](/img/fwupd-logo.svg)
+
+[fwupd](https://3mdeb.com/shop/services/basic-firmware-update-integration/)
+is an open-source daemon that manages the firmware updates of various
+devices. You will surely benefit from using fwupd if you are interested in
+automating the firmware update process on your devices and reducing the risk of
+security vulnerabilities. fwupd is compatible with a wide range of devices,
+including laptops, desktops, and IoT devices.
+
+Changes to fwupd can be divided into several areas. The first is the addition of
+[support for Qubes OS](https://github.com/fwupd/fwupd/pull/2710/commits/295418ef784408503b6a62991a74aa4bc886c4e0).
+These changes allow firmware updates from within one of the most
+secure operating systems. Norbert Kamiński was responsible for these changes.
```

Comment:
Link to profile.

The whole paragraph has a very generic description. It could be better and contain more meat, especially since you are the author of the changes.